tests/setup.py

```diff
@@ -135,7 +135,7 @@ def setUp(self) -> 'Generator':
         self.session = self.wm.get_session(self.config.name, self.view.file_name())
         self.assertIsNotNone(self.session)
         self.assertEqual(self.session.config.name, self.config.name)
-        yield {"condition": lambda: self.session.state == ClientStates.READY}
+        yield {"condition": (lambda: self.session.state == ClientStates.READY)}
```

Comment:
Because there's no `"timeout"` anymore, why not just write this as

```
yield lambda: self.session.state == ClientStates.READY`
```
(same for above)

Reply:
Oh, didn't mean to remove timeout, just PERIOD. But maybe we are OK without TIMEOUT?